web/src/components/BottomNavigation.tsx

```diff
@@ -71,7 +71,12 @@ const BottomNavigation = ({ cityModel, languageCode }: BottomNavigationProps): R
 
   return (
     <Container elevation={4}>
-      <StyledBottomNavigation id={BOTTOM_NAVIGATION_ELEMENT_ID} showLabels value={value} component='nav'>
+      <StyledBottomNavigation
+        id={BOTTOM_NAVIGATION_ELEMENT_ID}
+        showLabels
+        value={value}
+        component='nav'
+        role='navigation'>
```

Comment:
❓ Is it necessary to set the role to `navigation` on top of the component already being `nav`? https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Roles/navigation_role#description

> It is preferable to use the HTML5 nav element to define a navigation landmark. If the HTML5 nav element technique is not being used, use a role="navigation" attribute to define a navigation landmark.